### ISSUE TITLE
[kaeld on 2008-02-28 18:17:00] : StringHub - XML tag for LC source not parsed

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#801]
No associated GitHub commit

**Comment by dglo** [comment]
[kaeld on 2008-02-28 18:18:02]
  Fixed in rev 2664 on 2/22/2008.

**Comment by dglo** [comment]
[kaeld on 2008-04-04 16:19:13]
Just spoke with Dave Seckel who says that he finds IceTop hits with missing neighbors indicative of persistent problem in LC configuration

**Comment by dglo** [comment]
[kaeld on 2008-04-05 15:25:52]
Tested MPE LC setting on SPTS64 in run 3291 on April 5 15:15 UTC.  All DOMs on hub ichub21 were configured to have MPE triggers and MPE LC sources.  I dumped the monitoring files to search for records that the DOM had correctly received the command to set the LC source to MPE.  It appears that the DOMs are doing so - not sure why the LC source is updated twice - perhaps the run start 'reconfirms' this setting.  I have not as yet examined the data to look for missing LC neighbors.
